pkg/gui/gui.go

```diff
@@ -127,6 +121,20 @@ func (gui *Gui) scrollUpMain(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
+func (gui *Gui) handleScrollUpByViewName(viewName string) func(g *gocui.Gui, v *gocui.View) error {
```

Comment:
now that we're using this for the commit status panel, we should remove scrollUpMain and scrollDownMain and utilise this function instead